python/feathub/processors/flink/table_builder/tests/sliding_window_test_base.py

```diff
@@ -0,0 +1,45 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import datetime
+from typing import Dict
+
+from feathub.processors.flink.table_builder.tests.table_builder_test_base import (
+    FlinkTableBuilderTestBase,
+)
+
+
+class SlidingWindowTestBase(FlinkTableBuilderTestBase):
```

Comment:
Since there is only one subclass of `SlidingWindowTestBase`, do we still need this class?